app/screens/statistics-screen/statistics-rewarded-dashboard.tsx

```diff
@@ -94,15 +94,15 @@ export class StatisticsRewardedDashbaord extends React.Component<Props> {
     }
 
     const likeAmount = hasSelectedDayOfWeek
-      ? days[selectedDayOfWeek].totalLikeAmount
+      ? days[selectedDayOfWeek]?.totalLikeAmount || 0
       : weeklyLikeAmount
 
     const likeAmountFromCivicLikers = hasSelectedDayOfWeek
-      ? days[selectedDayOfWeek].totalCivicLikeAmount
+      ? days[selectedDayOfWeek]?.totalCivicLikeAmount || 0
       : weeklyLikeAmountFromCivicLikers
 
     const likeAmountFromCreatorFunds = hasSelectedDayOfWeek
-      ? days[selectedDayOfWeek].totalBasicLikeAmount
+      ? days[selectedDayOfWeek]?.totalBasicLikeAmount || 0
```

Comment:
Should the 0 be the guard for the whole ternary expression instead?

Reply:
Updated